zlrlo/210614-1.js

```diff
@@ -0,0 +1,33 @@
+function solution(record) {
+    let answer = [];
+    
+    let order = []; 
+    let map = {};
+    
+    record.forEach((data) => {
+        const splitData = data.split(' ');
+        
+        const state = splitData[0];
+        const id = splitData[1];
+        const nickname = splitData[2];
+        
+        order.push({state, id});   
+        
+        if(state === 'Enter' || state === 'Change') {
+            map[id] = nickname; 
+        }
```

Comment:
리뷰 연습) 
```suggestion
        if(state === 'Enter' || state === 'Change')  map[id] = nickname; 
```
이렇게 쓰면 더 간단하지 않을까요..?